src/UI/DappModal/dappModal.types.ts

```diff
@@ -15,6 +15,5 @@ export interface DappModalConfig {
 }
 
 export interface DappModalInteractionConfig {
-  openOnMount?: boolean;
```

Comment:
Is this not breaking?

Reply:
No, because we have the `visible` props/flag which was already doing that